tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -48,17 +50,20 @@ def tearDown(self):
         """At the end of test execution, it checks for the errors"""
         self.assertTrue(self.test_passed, msg=self.failcomment)
 
-    def take_screenshot(self):
-        """invokes screen capture event, crops the status bar, and saves to the file"""
+    # hook to override default locale setting
+    # see https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/gaia_test.py#L1019
+    def modify_settings(self, settings):
+        settings['language.current'] = self.locale
```

Comment:
I thought about adding it to the gaiatest too, but since this variable is already in testvars and I'm modifying it to use with imagecompare, I thought it might not be right to integrate it into gaiatest, since in runner/base.py, none of the parameters are about actually modifying a specific testvar value.  Since in most cases  we can easily change the locale by changing the testvar value (in RTL there is a potential need to run the same test in many different locale) perhaps we should see how this works out first.